pkg/parquet/search/parquet_queriable_test.go

```diff
@@ -0,0 +1,285 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/prometheus-community/parquet-common/blob/382b6ec8ae40fb5dcdcabd8019f69a4be1cd8869/search/parquet_queryable_test.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Prometheus Authors.
+
+package search
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/prometheus/prometheus/model/labels"
+	"github.com/prometheus/prometheus/promql"
+	"github.com/prometheus/prometheus/promql/promqltest"
+	prom_storage "github.com/prometheus/prometheus/storage"
+	"github.com/prometheus/prometheus/tsdb/chunkenc"
+	"github.com/prometheus/prometheus/util/teststorage"
+	"github.com/prometheus/prometheus/util/testutil"
+	"github.com/stretchr/testify/require"
+	"github.com/thanos-io/objstore/providers/filesystem"
+
+	"github.com/grafana/mimir/pkg/parquet/schema"
+	"github.com/grafana/mimir/pkg/parquet/storage"
+)
+
+func TestPromQLAcceptance(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Skipping, because 'short' flag was set")
+	}
```

Comment:
This test took ~15min in the CI. Is that a concern? Do we want to make it opt-in rather than opt-out? I'm mostly thinking about the impact on other mimir devs, not us

Reply:
Our CI health is already pretty terrible. This should be opt-in, 15m is a non-starter.

Reply:
It's fixed already to be opt-in with env var